README.md

```diff
@@ -1,6 +1,209 @@
-## 67P-Swarm
-Trajectory Optimization for a Spacecraft Swarm Orbiting 67P/Churyumov-Gerasimenko. 
+# Trajectory Optimization of a Spacecraft Swarm (TOSS)
 
-A thesis presented for the degree of Master of Science in mathematics, with specialization in Optimization and Systems theory provided by KTH Royal Institute of Technology. 
+<!-- TABLE OF CONTENTS -->
+<details open="open">
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+      <ul>
+        <li><a href="#built-with">Built With</a></li>
+      </ul>
+    </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#installation">Installation</a></li>
+        <li><a href="#test">Test</a></li>
+      </ul>
+    </li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#contributing">Contributing</a></li>
+    <li><a href="#license">License</a></li>
+    <li><a href="#contact">Contact</a></li>
+  </ol>
+</details>
 
-The thesis topic was provided by the Advanced Concepts Team (ACT) at The European Space Agency (ESA).  
+<!-- ABOUT THE PROJECT -->
+
+## About The Project
+
+Trajectory Optimization for a Spacecraft Swarm (TOSS) is a thesis project presented for the degree of Master of Science in mathematics, with specialization in Optimization and Systems theory provided by KTH Royal Institute of Technology. The thesis topic was defined within the scope of $\Phi$[-lab@Sweden](https://www.ai.se/en/data-factory/f-lab-sweden) in the frame of a collaboration between [AI Sweden](https://www.ai.se/en/) and the [European Space Agency](https://www.esa.int/) to explore distributed edge learning for space applications. The project was mainly supervised by Pablo Gómez and Emmanuel Blazquez from the Advanced Concepts Team (ACT) at ESA, and examined by Xiaoming Hu from KTH.  
+
+The goal of the project is to enable an evaluation of the effects for single and multi-spacecraft architectures when applied to a prospective mission with specified science objectives. In particular, TOSS is designed to be:
```

Comment:
Way too vague imo. You need to mention
- Optimizing trajectories with impulsive maneuvers around irregular bodies such as asteroids and comets using a polyhedral gravity model, RK xyz scheme and pygmo for evolutionary optimization.
- Goal of the trajectories is to cover the volume near the body well without getting too close

Reply:
Fixed!